src/components/Buttons/CloseChatWindowButton/CloseChatWindowButton.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CloseChatWindowButton from './CloseChatWindowButton';
+import CloseChatWindowIcon from '../../../icons/CloseChatWindowIcon';
+
+import useChatContext from '../../../hooks/useChatContext/useChatContext';
+
+jest.mock('../../../hooks/useChatContext/useChatContext');
+
+const mockUseChatContext = useChatContext as jest.Mock<any>;
+
+const mockToggleChatWindow = jest.fn();
+mockUseChatContext.mockImplementation(() => ({ setIsChatWindowOpen: mockToggleChatWindow }));
+
+describe('the CloseChatWindowButton component', () => {
+  it('should render correctly when chat window is open', () => {
+    const wrapper = shallow(<CloseChatWindowButton />);
+    expect(wrapper.containsMatchingElement(<CloseChatWindowIcon />)).toEqual(true);
```

Comment:
I don't think this test is needed. When shallow rendering this component, I think the icon will always be there regardless of the value of `isChatWindowOpen` (which is actually `undefined` here because it isn't defined on line 14. I think this is ok because this value is not relevant to this test.)